src/main/java/org/opentripplanner/routing/edgetype/StreetEdge.java

```diff
@@ -98,6 +99,12 @@ public NumericFieldSet getExtraNumericFields() {
     public void setExtraNumericFields(NumericFieldSet extraNumericFields) {
         this.extraNumericFields = extraNumericFields;
     }
+
+    public Date getLastAudited() {
```

Comment:
The bench and toilet counts aren't really NIH-specific, as they come from OSM data, and do not modify existing behavior unless extra query params are used.  They should probably get pulled out or moved eventually, anyways.